posts/maxistahl/01.open-ai-john-smith.md

```diff
@@ -0,0 +1,57 @@
+
+## Wrap up
+
+In conclusion, it was a pleasure to have the opportunity to interview John Smith, a senior software engineer at OpenAI, about his work and the exciting world of artificial intelligence. Through our conversation, we gained valuable insights into the daily responsibilities and challenges of working in the field of AI, as well as the potential impacts of AI on language, communication, and the human race.
+
+John shared with us his thoughts on the development of GPT-3 and other AI projects at OpenAI, and the ways in which AI could potentially change the way we communicate and interact with each other. He also highlighted the importance of approaching the development and use of AI with caution and consideration for its potential impacts and consequences, and ensuring that the benefits of AI are shared and distributed fairly across society.
+
+Overall, it was a thought-provoking and illuminating conversation that gave us a deeper understanding of the dynamic and rapidly-evolving field of artificial intelligence. Thank you, John, for sharing your expertise and insights with us!
+
+## The Impact
+
+John Smith is an avatar in a role game created with Assistant, the last version of GPT-3 from Open AI.
+
+This article is just an excercise to explore how do you feel about an entire article written by an AI created from a [dialogue](assets/open-ai-john-smith/play-role-playing-game.jpg) that we have. Maybe is not the first post, niether the last one created by an AI, but this kind of post going to be part of the near future.
+
+I have learned from this exercise that even while I was interacting with John (Assistant), the importance was not related about the depth of their answers nor to the technical aspect of this technology, but to the perceived value of the anwswers by a human (the meaning of a sentence, the choise of word over another, how a person feels about an idea, etc).
+
+An AI does not have the ability to explore the world (yet), so Many of the surprising answers it (he) gave me have a brief explanation of their nature: (I'm an AI, I have no opinion about...), what will happen in the world when an AI is able to learn from the context to perceive the value of what it learns? (not just be trained).
```

Comment:
Para que se entienda mejor, la reescribiría:

An AI does not have the ability to explore the world (yet), so many of the surprising answers it (he) gave me have a brief explanation of their nature: "I'm an AI, I have no opinion about...".
What will happen in the world when an AI can not only be trained, but also learn from its context to perceive the value of what it learns?

Reply:
got it, done